PanoptesNetClient/PanoptesNetClient/ApiClient.cs

```diff
@@ -32,23 +36,125 @@ public ApiClient(HttpClient client = null)
             }
         }
 
-        public async Task<JObject> GetAsync(IRequest request)
+        /// <summary>
+        /// Make a GET request for a single resource using IRequest
+        /// </summary>
+        #region Generic GET using IRequest
+        public async Task<T> Get<T>(IRequest request)
         {
-            JObject resource = null;
+            List<T> collection = await GetAsync<T>(request.Endpoint);
 
-            HttpResponseMessage response = await Client.GetAsync(request.Endpoint);
+            if (collection.Count > 0)
+            {
+                return collection[0];
+            }
+            return default(T);
+        }
+        #endregion
+
+        /// <summary>
+        /// Make a GET request for a single resource using an endpoint
+        /// </summary>
+        #region Generic GET using string
+        public async Task<T> Get<T>(string route)
```

Comment:
For all of these functions that use generics I think it would be better to use `IResource` instead of `T`